frontend/src/employee-frontend/components/payments/Actions.tsx

```diff
@@ -53,25 +60,52 @@ const Actions = React.memo(function Actions({
           <Gap size="s" horizontal />
         </>
       ) : null}
-      <AsyncButton
-        text={i18n.payments.buttons.deletePayment(checkedIds.length)}
-        disabled={checkedIds.length === 0}
-        onClick={() => deleteDraftPaymentsResult({ body: checkedIds })}
-        onSuccess={() => {
-          actions.clearChecked()
-          reloadPayments()
-        }}
-        data-qa="delete-payments"
-      />
-
-      <Gap size="s" horizontal />
-      <Button
-        primary
-        disabled={checkedIds.length === 0}
-        text={i18n.payments.buttons.sendPayments(checkedIds.length)}
-        onClick={actions.openModal}
-        data-qa="open-send-payments-dialog"
-      />
+      {status === 'DRAFT' ? (
+        <>
+          <AsyncButton
+            text={i18n.payments.buttons.deletePayment(checkedIds.length)}
+            disabled={checkedIds.length === 0}
+            onClick={() => deleteDraftPaymentsResult({ body: checkedIds })}
+            onSuccess={() => {
+              actions.clearChecked()
+              reloadPayments()
+            }}
+            data-qa="delete-payments"
+          />
+          <Gap size="s" horizontal />
+          <AsyncButton
+            text={i18n.payments.buttons.confirmPayments(checkedIds.length)}
+            disabled={checkedIds.length === 0}
+            onClick={() => confirmDraftPaymentsResult({ body: checkedIds })}
+            onSuccess={() => {
+              actions.clearChecked()
+              reloadPayments()
+            }}
+            data-qa="confirm-payments"
+          />
+        </>
+      ) : status === 'CONFIRMED' ? (
+        <>
+          <AsyncButton
+            text={i18n.payments.buttons.revertPayments(checkedIds.length)}
+            disabled={checkedIds.length === 0}
+            onClick={() => revertPaymentsResult({ body: checkedIds })}
+            onSuccess={() => {
+              actions.clearChecked()
+              reloadPayments()
+            }}
+            data-qa="revert-payments"
+          />
```

Comment:
FWIW you could probably also use `MutateButton` and skip the legacy `wrapResult` if you wanted to make this more up to date while at it